cmake/toolchains/libfuzzer.cmake

```diff
@@ -1,2 +1,7 @@
-# Require libfuzzer specific flags
-set(CMAKE_CXX_FLAGS "-O1 -gline-tables-only -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -stdlib=libstdc++ -fopenmp=libgomp")
+# Inherit default options
+include("${CMAKE_CURRENT_LIST_DIR}/default.cmake")
+# Disable Z3 and CVC4 since none of the existing fuzzers need them
+set(USE_Z3 OFF)
```

Comment:
This might need to be something like ``set(USE_Z3 OFF CACHE BOOL "" FORCE)`` to work, but it might be fine as is, we need to check.

Reply:
You are right, simply doing ``set(USE_Z3 OFF)`` did not work as expected.